pkg/dockerclient/shell.go

```diff
@@ -241,9 +241,33 @@ func (c *ShellClient) RunContainer(imageName string, runOptions *RunOptions) (st
 		}
 	}
 
+	userArgument := ""
+	if runOptions.RunAsUser != nil || runOptions.RunAsGroup != nil {
+		userStr := ""
+		if runOptions.RunAsUser != nil && runOptions.RunAsGroup != nil {
+
+			// user and group
+			userStr = fmt.Sprintf("%d:%d", runOptions.RunAsUser, runOptions.RunAsGroup)
+		} else if runOptions.RunAsUser != nil {
+
+			// only user
+			userStr = fmt.Sprintf("%d", runOptions.RunAsUser)
+		} else {
+
+			// only group
+			userStr = fmt.Sprintf(":%d", runOptions.RunAsGroup)
+		}
```

Comment:
simplify:
```go
userStr := ""
	if runOptions.RunAsUser != nil && runOptions.RunAsGroup != nil {
		if runOptions.RunAsUser != nil {
			userStr = fmt.Sprintf("%d", runOptions.RunAsUser)
		}
		if runOptions.RunAsUser != nil {
			userStr += fmt.Sprintf(":%d", runOptions.RunAsGroup)
		}
	...
```

Reply:
The snippet you left doesn't really work, because if `RunAsUser != nil` and `RunAsGroup != nil` from the outer `if` then the inner `if`s are redundant. But I'll try to simplify the logic anyway